data/versioned-api/crud-api-version-1.json

```diff
@@ -0,0 +1,1106 @@
+          "arguments": {
+            "filter": {
+              "_id": 7
+            }
+          }
+        }
+      ],
+      "expectEvents": [
+        {
+          "client": "client",
+          "events": [
+            {
+              "commandStartedEvent": {
+                "command": {
+                  "delete": "test",
+                  "deletes": [
+                    {
+                      "q": {
+                        "_id": 7
+                      },
+                      "limit": 1
+                    }
+                  ],
+                  "apiVersion": "1",
+                  "apiStrict": {
+                    "$$unsetOrMatches": false
+                  },
+                  "apiDeprecationErrors": true
+                }
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "description": "distinct appends declared API version",
+      "operations": [
+        {
+          "name": "distinct",
+          "object": "collection",
+          "arguments": {
+            "fieldName": "x",
+            "filter": {}
+          }
+        }
+      ],
+      "expectEvents": [
+        {
+          "client": "client",
+          "events": [
+            {
+              "commandStartedEvent": {
+                "command": {
+                  "distinct": "test",
+                  "key": "x",
+                  "apiVersion": "1",
+                  "apiStrict": {
+                    "$$unsetOrMatches": false
+                  },
+                  "apiDeprecationErrors": true
+                }
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "description": "estimatedDocumentCount appends declared API version on 4.9.0 or greater",
+      "operations": [
+        {
+          "name": "estimatedDocumentCount",
+          "object": "collection",
+          "arguments": {}
+        }
+      ],
+      "expectEvents": [
+        {
+          "client": "client",
+          "events": [
+            {
+              "commandStartedEvent": {
+                "command": {
+                  "aggregate": "test",
+                  "pipeline": [
+                    {
+                      "$collStats": {
+                        "count": {}
+                      }
+                    },
+                    {
+                      "$group": {
+                        "_id": 1,
+                        "n": {
+                          "$sum": "$count"
+                        }
+                      }
+                    }
+                  ],
+                  "apiVersion": "1",
+                  "apiStrict": {
+                    "$$unsetOrMatches": false
+                  },
+                  "apiDeprecationErrors": true
+                }
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "description": "find command with declared API version appends to the command, but getMore does not",
```

Comment:
This may be out of date and need to be resynced to get tests passing.

Reply:
Resynced all tests one more time!